packages/desktop-client/src/components/manager/subscribe/Bootstrap.tsx

```diff
@@ -56,19 +58,25 @@ export function Bootstrap() {
 
   return (
     <View style={{ maxWidth: 450, marginTop: -30 }}>
-      <Title text="Welcome to Actual!" />
+      <Title text={t('Welcome to Actual!')} />
       <Paragraph style={{ fontSize: 16, color: theme.pageTextDark }}>
-        Actual is a super fast privacy-focused app for managing your finances.
-        To secure your data, you’ll need to set a password for your server.
+        {t(
+          'Actual is a super fast privacy-focused app for managing your finances.',
+        )}
+        {t(
+          'To secure your data, you’ll need to set a password for your server.',
+        )}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think it makes more sense for this to be a single translation string since it's a single block on content. Also: using `Trans` over `t` in this instance.